client/app/bundles/course/assessment/submission/containers/VoiceResponseAnswer.jsx

```diff
@@ -101,9 +110,7 @@ class VoiceResponseAnswer extends Component {
   };
 
   renderAudio = (field) => {
-    const {
-      input: { value },
-    } = field;
+    const { value } = field;
     const { file, url } = value;
```

Comment:
I believe this can be condensed into a single line, something like this:

```js
const { value: { file, url } } = field;
```

Need to check.